modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator_{context}"]
+= Installing the AWS Load Balancer Operator
+
+You can install the AWS Load Balancer Operator using the Red Hat {product-title} Operator Hub.
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+.Procedure
+
+. Click *Operators* → *OperatorHub* in the OpenShift Web Console.
+. Click *AWS Load Balancer Operator*.
+You can use the Filter by keyword text box or the filter list to search for AWS Load Balancer Operator from the list of operators.
```

Comment:
*Filter by keyword* text box

list of Operators per  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization

Reply:
Ack'ed